src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -263,7 +260,7 @@ private function addTemplatingSection(ArrayNodeDefinition $rootNode)
                                     ->addDefaultChildrenIfNoneSet()
                                     ->prototype('scalar')->defaultValue('FrameworkBundle:Form')->end()
                                     ->validate()
-                                        ->ifTrue(function ($v) {return !in_array('FrameworkBundle:Form', $v); })
+                                        ->ifNotInArray(array('FrameworkBundle:Form'))
```

Comment:
These changes are wrong. You are now checking whether ``$v`` is in the array ``array('FrameworkBundle:Form')`` (which will never be the case as ``$v`` is an array) while the old code was checking whether ``'FrameworkBundle:Form'`` is in the ``$v`` array.